src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,51 @@
 package mate.academy;
 
+import java.time.LocalDateTime;
+import java.util.List;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
     public static void main(String[] args) {
-        MovieService movieService = null;
+        Injector injector = Injector.getInstance("mate.academy");
+
+        MovieService movieService =
+                (MovieService) injector.getInstance(MovieService.class);
 
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
         movieService.add(fastAndFurious);
         System.out.println(movieService.get(fastAndFurious.getId()));
         movieService.getAll().forEach(System.out::println);
+
+        CinemaHallService cinemaHallService =
+                (CinemaHallService) injector.getInstance(CinemaHallService.class);
+
+        CinemaHall cinemaHall = new CinemaHall();
+        cinemaHall.setCapacity(200);
+        cinemaHall.setDescription("CinemaTheatre in Ocean Plaza");
+        cinemaHallService.add(cinemaHall);
+        System.out.println(movieService.get(cinemaHall.getId()));
+        movieService.getAll().forEach(System.out::println);
+
+        LocalDateTime datetime = LocalDateTime.of(2022, 1, 2, 10, 15);
+        MovieSession movieSession = new MovieSession();
+        movieSession.setMovie(fastAndFurious);
+        movieSession.setCinemaHall(cinemaHall);
+        movieSession.setShowTime(datetime);
+        MovieSessionService movieSessionService =
+                (MovieSessionService) injector.getInstance(MovieSessionService.class);
+
+        movieSessionService.add(movieSession);
+        System.out.println(movieSessionService.get(movieSession.getId()));
+        List<MovieSession> availableSessions = movieSessionService
+                .findAvailableSessions(fastAndFurious.getId(), datetime.toLocalDate());
+        System.out.println(availableSessions.get(0).getCinemaHall().getDescription());
```

Comment:
I think better test this method using
```suggestion
        movieSessionService
                .findAvailableSessions(fastAndFurious.getId(), LocalDate.now())
                .forEach(System.out::println);
```